src/client/dfuse/dfuse_main.c

```diff
@@ -242,6 +242,7 @@ show_help(char *name)
 		"	-S --singlethread	Single threaded\n"
 		"	-t --thread-count=count	Number of fuse threads to use\n"
 		"	-f --foreground		Run in foreground\n"
+		"	   --enable-caching	Enable all caching\n"
```

Comment:
Should there be an entry for --enable-wb-cache here?

I also wonder if the options that are on by default should indicate that?
```suggestion
		"	   --enable-caching	Enable all caching (default)\n"
		"	   --enable-wb-caching	Use write-back cache rather than write-through (default)\n"
```

Reply:
Yes I like the default for both text for both of those. I'll add "enable-wb-cache" instead of "enable-wb-caching" to match the disable tense of "disable-wb-cache".